openapi.yaml

```diff
@@ -30,11 +30,17 @@ paths:
         - $ref: '#/components/parameters/pageNumber'
         - $ref: '#/components/parameters/pageSize'
         - in: query
-          name: species
+          name: filter[species]
           description: Search by pet species
           required: false
           schema:
             type: string
+        - in: query
+          name: filter[isCat]
```

Comment:
Although this is just a skeleton code, I guess this filter is overlapped with `filter[species]=cat`. I'd suggest using other parameters e.g. `filter[hasOwner]` and add a few more pets without owners for testing purpose.